db/schema.rb

```diff
@@ -10,8 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 0) do
+ActiveRecord::Schema[7.0].define(version: 2022_05_17_205718) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "users", force: :cascade do |t|
```

Comment:
Looks like you created the users table, but no matching model on `app/models/user.rb`. maybe you missed that file?

Reply:
@diego-suarez I Changed the gem in the last commit.